Docker/README.md

```diff
@@ -1,75 +1,19 @@
 # Docker deployment instructions
```

Comment:
Why are we getting rid of this?

Reply:
Oops didn’t pull before pushing! One second

Reply:
No problem <3